### PR TITLE
fmtowns_cd.xml: 21 new dumps, 21 replacements

### PR DESCRIPTION
**Pull request by rzero9**
New working software list additions
-----------------------------------

CRISS Ver. 1.00 [redump.org]
Demon City [redump.org]
Dengeki Nurse 2 - More Sexy [redump.org, wiggy2k]
Free Software Collection 5 (HMD-117A) [redump.org]
Free Software Collection 5 (HMD-117B) [redump.org]
Free Software Collection 6 (HME-141A) [redump.org]
Free Software Collection 7 (HME-225A) [redump.org]
Hacchake Ayayo-san IV - Sexy Olympic / Ayayo's Live Affection
[redump.org]
if 3 - Invitations from Fantastic Stories [redump.org]
Kyouko no Ijiwaru!! Hachamecha Daishingeki [redump.org]
New 3D Golf Simulation - Eight Lakes G.C. [redump.org]
Only You - Seikimatsu no Juliet-tachi [redump.org]
Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku [redump.org]
Pi's Solitaire Royale + Tom Snyder's Puppy Love [redump.org]
Sayonara no Mukougawa [redump.org]
Terra Towns [redump.org]
The Queen of Duellist Gaiden + Gaiden Alpha (1994-03-02) [redump.org,
wiggy2k]
WonPara Wars II [redump.org]

New not working software list additions
---------------------------------------

Custom Mate + Denwa no Bell ga... [redump.org]
Sangokushi IV [redump.org]
Toushin Toshi II (1995-04-26) [redump.org]

Replaced software list items
----------------------------

The 4th Unit 1-2 Towns - Linkage [redump.org]
The 4th Unit 3 - Dual Targets [redump.org]
Angel Halo [redump.org]
Appare CD Vol. 2 - Houou no Maki [redump.org]
Bell's Avenue Vol. 1 [redump.org]
D.P.S. Zenbu [redump.org]
Daikoukai Jidai II [redump.org]
De.FaNa [redump.org]
Desire - Haitoku no Rasen [redump.org]
Doukyuusei [redump.org]
Doukyuusei 2 [redump.org]
Etsuraku no Gakuen [redump.org]
F29 Retaliator [redump.org, wiggy2k]
Gakuen King - Hidehiko Gakkou o Tsukuru [redump.org]
Hatchake Ayayo-san 1-2-3 [redump.org]
Mandala-ke Ichizoku [redump.org]
Monkey Island 2 - LeChuck's Revenge [redump.org]
Mugen Houyou [redump.org]
Rance 4.1 - Okusuri Koujou o Sukue! [redump.org]
Rance 4.2 - Angel-gumi [redump.org]
The Date - Kore de Kanojo wa Boku no Mono! [redump.org]

### CODE REVIEW

File: hash/fmtowns_cd.xml [hunk 1]
```diff
@@ -12508,6 +12983,47 @@ User/save disks that can be created from the game itself are not included.
...
+		<rom name="Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku (Japan) (Track 20).bin" size="2476656" crc="295c363e" sha1="31247daf79c1e445163fd17b664d33fcfb3de690"/>
+		<rom name="Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku (Japan) (Track 21).bin" size="25048800" crc="2a8209e9" sha1="72fed63208441de7b4d795187d139a83b905737b"/>
+		<rom name="Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku (Japan) (Track 22).bin" size="31763760" crc="afe7543b" sha1="9d1a71beb5da628205fb76f758132451e1b19879"/>
+		<rom name="Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku (Japan).cue" size="3436" crc="0c266be6" sha1="ca563ab2c656ebf280b6a3317f8eb24620d41de9"/>
+		-->
+		<description>Moeru Asoko no Paipai Yuugi - Fukkatsu! Mahjong Oukoku</description>
+		<year>1993</year>
+		<publisher>イリュージョン (Illusion)</publisher>
+		<info name="serial" value="HME-252"/>
+		<info name="alt_title" value="燃えるあそこの牌牌遊戯　復活！まぁじゃん王国" />
```
Comment (cuavas) [inline hash/fmtowns_cd.xml:13017]: Is “まぁじゃん王国” correct?  Small kana are used for vowel substitution, not prolonged vowels, and substituting “a” into “ma” doesn’t really make sense if you try to read it as-is.

Comment (einstein95) [reply hash/fmtowns_cd.xml:13017]: Yes, the [title screen](https://refuge.tokyo/pc9801/pc98/screenshot/04343.png) uses "まぁじゃん王国"

Comment (cuavas) [reply hash/fmtowns_cd.xml:13017]: Probably worth adding a comment so it doesn’t get “corrected” like “kernal” and “Shangrlia”.

Comment (rzero9) [reply hash/fmtowns_cd.xml:13017]: Added a comment, just in case.

**Comment by rzero9** [comment]
Sorry for the additional commits, I had to fix some minor mistakes. It should be ready to merge now.

**Pull request merged**